packages/fluentui/react-northstar-prototypes/src/prototypes/FormValidation/index.tsx

```diff
@@ -1,19 +1,10 @@
 import * as React from 'react';
 import { PrototypeSection, ComponentPrototype } from '../Prototypes';
-import FormValidateFormik from './Formik';
 import FormValidateHooks from './FormHooks';
 
 const FormValidationPrototype: React.FC = () => {
   return (
     <>
-      <PrototypeSection title="Formik">
-        <ComponentPrototype
-          title="List with context menu"
-          description="Context menu can be opened by clicking on the more button or by right mouse button"
-        >
-          <FormValidateFormik />
```

Comment:
can you replace it with something like

```
<iframe src="https://codesandbox.io/embed/fluent-ui-example-forked-hgp51?fontsize=14&hidenavigation=1&theme=dark"
     style="width:100%; height:500px; border:0; border-radius: 4px; overflow:hidden;"
     title="fluent-ui-example (forked)"
     allow="accelerometer; ambient-light-sensor; camera; encrypted-media; geolocation; gyroscope; hid; microphone; midi; payment; usb; vr; xr-spatial-tracking"
     sandbox="allow-forms allow-modals allow-popups allow-presentation allow-same-origin allow-scripts"
   ></iframe>
```

Reply:
This is now fixed, undid the change here and bumped formik to a build that makes everything happier.